src/parsers/sample-models.hxx

```diff
@@ -154,9 +154,9 @@ namespace pinocchio
         
         parentFrame = model.getBodyId(pre+"wrist1_body");
         GeometryObject wristBall(pre+"wrist_object",
-                                 parentFrame, model.frames[parentFrame].parent,
-                                 boost::shared_ptr<fcl::Sphere>(new fcl::Sphere(0.05)),
+                                 model.frames[parentFrame].parentJoint, parentFrame,
                                  SE3::Identity(),
+                                 boost::shared_ptr<fcl::Sphere>(new fcl::Sphere(0.05)),
```

Comment:
Coming here a bit late but still worth for future changes. It is better to use `hpp::fcl::shared_ptr` for hpp-fcl objects.